### PR TITLE
flexible schema binding variable for DDL generation; script alignment; backup database script

### PR DESCRIPTION
**Pull request by jdide**

### CODE REVIEW

File: R/ResultsDataModel.R [hunk 1]
```diff
@@ -21,12 +21,14 @@
...
 #' @param packageName    The name of the R package whose output model we are documenting.
+#' @param schemaBindingVariable     The name of the schema binding parameter
 #' 
 #' @export
 createDdl <- function(packageName,
                       packageVersion,
                       modelVersion,
-                      specification){
+                      specification,
+                      schemaBindingParam = "databaseSchema"){
```
Comment (gowthamrao) [inline R/ResultsDataModel.R:31]: should be `schemaBindingVariable = "databaseSchema"` based on documenation?

Comment (gowthamrao) [reply R/ResultsDataModel.R:31]: We could call it is as 'databaseSchema' to represent a generic database + schema combination

**Review by gowthamrao — CHANGES_REQUESTED** [review]
Please see request for small changes

File: R/ResultsDataModel.R (hunk 1 above)
Comment (jdide) [reply R/ResultsDataModel.R]: > should be `schemaBindingVariable = "databaseSchema"` based on documenation?

good catch @gowthamrao, can you please review again?

**Pull request closed without merging**